<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[TestcreateLeadTestis FAIL/SKIP]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="79397" started-at="2019-11-07T12:48:00Z" finished-at="2019-11-07T12:49:19Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="79397" started-at="2019-11-07T12:48:00Z" finished-at="2019-11-07T12:49:19Z">
      <class name="qa.scripts.CreateLead">
        <test-method status="PASS" signature="openApp()[pri:0, instance:qa.scripts.CreateLead@43a25848]" name="openApp" is-config="true" duration-ms="67020" started-at="2019-11-07T18:18:00Z" finished-at="2019-11-07T18:19:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openApp -->
        <test-method status="FAIL" signature="createLeadTest()[pri:0, instance:qa.scripts.CreateLead@43a25848]" name="createLeadTest" duration-ms="11643" started-at="2019-11-07T18:19:07Z" finished-at="2019-11-07T18:19:18Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at qa.pom.lead.CreateLeadPage.clickcampSrc(CreateLeadPage.java:126)
	at qa.scripts.CreateLead.createLeadTest(CreateLead.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createLeadTest -->
        <test-method status="PASS" signature="closeApp(org.testng.ITestResult)[pri:0, instance:qa.scripts.CreateLead@43a25848]" name="closeApp" is-config="true" duration-ms="673" started-at="2019-11-07T18:19:18Z" finished-at="2019-11-07T18:19:19Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=createLeadTest status=FAILURE method=CreateLead.createLeadTest()[pri:0, instance:qa.scripts.CreateLead@43a25848] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[TestcreateLeadTestis FAIL/SKIP]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeApp -->
      </class> <!-- qa.scripts.CreateLead -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
